docs/tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     /* files are emitted by babel */
     "noEmit": true,
-    "types": ["react", "node"],
+    "types": ["react", "node", "requestidlecallback"],
```

Comment:
Used https://github.com/jsprateek/wordress-projects/blob/a7fb6bbf56089eb3a181d7ad7bb2b241e3d4e77b/packages/priority-queue/tsconfig.json#L6 as inspiration for the solution